apps/session-manager/src/repositories/fast-login-api.ts

```diff
@@ -0,0 +1,24 @@
+import * as nodeFetch from "node-fetch";
+import { createClient, Client } from "../generated/fast-login-api/client";
+
+export function getFastLoginLollipopConsumerClient(
+  token: string,
+  baseUrl: string,
+  basePath?: string,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  fetchApi: typeof fetch = nodeFetch as unknown as typeof fetch,
```

Comment:
Add a TODO "customize fetch with timeout"

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/18/commits/8a083e326d241baa8febfd77f182b4588be698bc